src/main/java/core/basesyntax/services/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.interfaces.FileWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    private static final String CREATE_FILE_EXCEPTION_MESSAGE = "Can't create new file ";
+    private static final String WRITE_FILE_EXCEPTION_MESSAGE = "Can't write file";
+
+    @Override
+    public boolean write(String dataToWrite, String fileName) {
+        File file = new File(fileName);
+        if (!file.exists()) {
```

Comment:
Here same as in FileReader

Reply:
I create a new file for report, if it not exists. I would have exception, if file is not exists